Increase the stack size to 8kB.  This is only required for i386 arch,
because other kernels always have 8kB stack sizes.

Index: linux-2.6.9-5.0.3.EL/include/asm-i386/thread_info.h
===================================================================
--- linux-2.6.9-5.0.3.EL.orig/include/asm-i386/thread_info.h	2005-02-25 10:25:33.000000000 +0200
+++ linux-2.6.9-5.0.3.EL/include/asm-i386/thread_info.h	2005-02-25 20:19:11.676139032 +0200
@@ -54,7 +54,7 @@
 
 #define PREEMPT_ACTIVE		0x10000000
 #ifdef CONFIG_4KSTACKS
-#define THREAD_SIZE            (4096)
+#define THREAD_SIZE            (8192)
 #else
 #define THREAD_SIZE		(8192)
 #endif
